src/kontrol/solc_to_k.py

```diff
@@ -954,7 +954,7 @@ def method_sentences(self) -> list[KSentence]:
 
     @property
     def sentences(self) -> list[KSentence]:
-        return [self.subsort, self.production, self.macro_bin_runtime, self.macro_init_bytecode] + self.method_sentences
+        return [self.subsort, self.production] + self.method_sentences
```

Comment:
How come these went away?

Reply:
Those macro's are looking like this
```
rule  ( #binRuntime ( S2KlibZModforgeZSubstdZModsrcZModScriptBase ) => #parseByteStack ( "0x..." ) )
```
and we initialised a kcfg of a given proof with 
```
<program> #binRuntime(contract.name_with_path)</program>
<jumpDests> #computeJumpDests(#binRuntime(contract.name_with_path)) </jumpdests>
```
So, with these macros I wasn't able to compute the jumpdests using the implementation from https://github.com/runtimeverification/evm-semantics/pull/2441.
Now, I initialise the kcfg with the bytecode directly. As a consequence, those macros are no longer used and so I thought it is a good idea to remove them from the generated k code.

Reply:
Another option, if we decide to keep the macro's, is to set the `<k>` cell to `#load_program ~> execute` instead of just `#execute`.

Reply:
Since they do not appear to be needed for anything else, I am happy with removing them. If we need to bring them back for some reason, we can.